README.md

```diff
@@ -158,11 +156,13 @@ function select(state) {
 export default class CounterApp {
   render() {
     return (
-      <Connector select={select}>
-        {({ counter, dispatch }) =>
+      // Passing an object of action creators here automatically binds them 
+      // with Redux's dispatcher and passes them to the child function
+      // (so they can be passed as props to child components)
+      <Connector select={select} actionCreators={CounterActions}>
+        {({ counter, actions }) =>
```

Comment:
Quick question, isn't `select` a bit too generic? Flummox is calling the `stores` and `actions`, I think the distinction is worth it. The way I see it, it makes the overall code simpler to read and follow. Am I missing something and is there any specific reasons for the current names?